### DISCUSSION TITLE
Consulta KD-Tree

### DISCUSSION THREAD

**Discussion by MaximilianWilke** [IIC2133-PUC/2024-2#140] [Tarea 2] [answered]
Hola, estuve investigando acerca e la implementación de un arbol cono K dimensiones, mi consulta es sobre si al insertar un nodo, se debe realizar algun tipo de balance? En las paginas que he revisado no se menciona esto pero quería estar seguro

**Comment by sebacarrascop** [comment, marked as answer]
hola!, cuando insertas un nodo en un KD-Tree, simplemente se ubica en la posición correspondiente siguiendo las reglas del árbol, sin realizar rotaciones.

**Reply by catalinacifuentes** [reply]
Hola! @sebacarrascop Me queda una duda, no es necesario balancear el KD Tree entonces? O insertar los pinguinos de forma que no quede desbalanceado?

**Comment by MaximilianWilke** [comment]
Y recomiendan empezar implementando KD-Tree desde la 1.1?

**Reply by sebacarrascop** [reply]
Hola!, esas son decisiones de diseño propias, te recuerdo que puedes usar diferentes arboles para abordar cada consulta, no hay necesidad de usar un único árbol para todas.
